styles/cspace-ui/Logo.css

```diff
@@ -1,3 +1,14 @@
+@value textDark from '../colors.css';
+@value activeEdgeWidth from '../dimensions.css';
+
+.common > a:focus {
+  outline: solid textDark;
+  outline-width: activeEdgeWidth;
+  outline-offset: activeEdgeWidth;
+  border-radius: 1px;
+  position: relative;
+}
+
```

Comment:
There is already styling for `a:focus` in RootPage.css (a thin dotted rectangular outline), which I'd like to just inherit for this. What you've added looks more like the styling that's intended for input fields and buttons (or links that look like buttons).

Reply:
Yea I wasn't sure which focus style to go with, so went with the more bold one. I just updated to be aligned with the other anchor elements.